nextflow.config

```
@@ -131,8 +135,19 @@ profiles {
         charliecloud.enabled    = false
         apptainer.enabled       = false
     }
-    arm {
-        docker.runOptions       = params.use_gpu ? '-u $(id -u):$(id -g) --platform=linux/amd64 --gpus all' : '-u $(id -u):$(id -g) --platform=linux/amd64'
+    arm64 {
+        process.arch            = 'arm64'
+        // TODO https://github.com/nf-core/modules/issues/6694
+        // For now if you're using arm64 you have to use wave for the sake of the maintainers
+        // wave profile
+        apptainer.ociAutoPull   = true
+        singularity.ociAutoPull = true
+        wave.enabled            = true
+        wave.freeze             = true
+        wave.strategy           = 'conda,container'
+    }
+    emulate_amd64 {
+        docker.runOptions       = '-u $(id -u):$(id -g) --platform=linux/amd64'
```

you should keep the line you had before for arm here
```suggestion
        docker.runOptions       = params.use_gpu ? '-u $(id -u):$(id -g) --platform=linux/amd64 --gpus all' : '-u $(id -u):$(id -g) --platform=linux/amd64'
```